packages/phrases/src/locales/zh-cn/translation/admin-console/applications.ts

```diff
@@ -34,12 +34,28 @@ const applications = {
     },
   },
   guide: {
+    header_title: 'Select a framework or tutorial', // UNTRANSLATED
```

Comment:
Do we need to translate those keys by gpt? Not sure about the convention.

Reply:
Yes, will translate in a follow-up PR